lib/github_workflow/cli.rb

```diff
@@ -11,6 +11,7 @@ module GithubWorkflow
   class Cli < Thor
     PROCEED_TEXT = "Proceed? [y,yes]: ".freeze
 
+    CONFIG_FILE_NAME =".github_workflow".freeze
```

Comment:
suggestion(blocking): Add space after constant and space after `=`

```suggestion
    CONFIG_FILE_NAME = ".github_workflow".freeze

```

Reply:
:facepalm: